src/utils/__tests__/noop.test.ts

```diff
@@ -14,11 +14,10 @@
  * limitations under the License.
  */
 
-import { expect } from "chai";
 import noop from "../noop";
 
 describe("utils - noop", () => {
     it("should do nothing at all", () => {
-        expect(noop()).to.equal(undefined);
+        expect(noop()).toBe(undefined);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this use of the output from "noop"; "noop" doesn't return anything. ([typescript:S3699](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS3699))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=440&issues=AWgpL78NUMkE2J58feAa&open=AWgpL78NUMkE2J58feAa)

[ISSUE_KEY]: AWgpL78NUMkE2J58feAa